views/map/index.android.js

```diff
@@ -29,8 +29,7 @@ export default class OlafMapView extends React.Component {
           <Button
             onPress={() => Linking.openURL(mapInfo.url).catch(err => console.error('An error occurred', err))}
             style={styles.button}
-          >
-          See the map!</Button>
+          > View Map </Button>
```

Comment:
The spaces, oddly, were keeping it from cutting off the text. I don't understand it, but it works.